index.js

```diff
@@ -529,7 +529,7 @@ function getHeaders (opts) {
   const headers = {}
   if (opts.secretToken) headers['Authorization'] = 'Bearer ' + opts.secretToken
   headers['Accept'] = 'application/json'
-  headers['User-Agent'] = opts.userAgent + ' ' + pkg.name + '/' + pkg.version
+  headers['User-Agent'] = `${opts.userAgent} ${pkg.name}/${pkg.version}/${process.release.name}/${process.versions.node}`
```

Comment:
There should be a space before `${process.release.name}`, not a slash. We want the user agent to be something like `apm-nodejs-agent/1.2.3 apm-nodejs-http-client/1.2.3 node/1.2.3`. The first part comes from the agent, the second part is the `${pkg.name}/${pkg.version}` part, and we want to add is another name/version pair for the Node.js runtime.

Reply:
Thanks for the explanation
I hope is ok now :)